Bootstrap fails on fresh Ubuntu 13.04

I set up a fresh Ubuntu 13.04 as recommended but running the bootstrap script produces:

```
....
Unpacking pluginhook (from .../tmp/pluginhook_0.1.0_amd64.deb) ...
Setting up pluginhook (0.1.0) ...
lsmod | grep aufs || modprobe aufs || apt-get install -y linux-image-extra-`uname -r`
apt-add-repository -y ppa:dotcloud/lxc-docker
make: apt-add-repository: Command not found
make: *** [docker] Error 127
```

And it seems the `apt-add-repository` command isn't there. Is it suppose to be in Ubuntu 13.04? This is a fresh install using the VMware Fusion Easy Install.

--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -3,7 +3,7 @@ export DEBIAN_FRONTEND=noninteractive
 export DOKKU_REPO=${DOKKU_REPO:-"https://github.com/progrium/dokku.git"}
 
 apt-get update
-apt-get install -y git make curl
+apt-get install -y git make curl software-properties-common
 
 cd ~ && test -d dokku || git clone $DOKKU_REPO
 cd dokku && test $DOKKU_BRANCH && git checkout origin/$DOKKU_BRANCH || true

